resources/sql/application-queries.sql

```diff
@@ -6,32 +6,41 @@ insert into applications (form_id, key, content, lang, preferred_name, last_name
 select a.id,
   a.key, a.lang,
   a.preferred_name || ' ' ||  a.last_name as applicant_name,
-  a.modified_time, coalesce(ar.state, 'received') as state
+  a.created_time, coalesce(ar.state, 'received') as state
 from applications a
 left outer join application_reviews ar on a.id = ar.application_id
 join forms f on f.id = a.form_id and f.key = :form_key
-order by a.modified_time desc;
+order by a.created_time desc;
 
 -- name: yesql-get-application-events
-select event_type, time, new_review_state from application_events where application_id = :application_id;
+select event_type, time, new_review_state, application_key, id from application_events where application_key = :application_key;
 
 -- name: yesql-get-application-review
-select id, application_id, modified_time, state, notes from application_reviews where application_id = :application_id;
+select id, application_id, modified_time, state, notes, application_key from application_reviews where application_key = :application_key;
 
 -- name: yesql-application-query-by-modified
```

Comment:
This is probably obsolete now and should be removed because it returns all the versions of the application. Excel export still used this, so it has to be fixed.